src/Categories/Examples/Monad/Duration.agda

```diff
@@ -0,0 +1,99 @@
+{-# OPTIONS --safe --without-K #-}
+module Duration where
+
+open import Level
+
+open import Function using (_$_)
+open import Function.Equality using (Π; _⟨$⟩_; cong)
+
+open import Relation.Binary
+open import Relation.Binary.Indexed.Heterogeneous.Construct.Trivial
+
+open import Data.Product using (Σ; _,_; _×_; proj₁; proj₂)
+open import Algebra using (Monoid; IsMonoid; IsSemigroup; IsMagma)
+
+open import Categories.Category
+open import Categories.Category.Instance.Setoids
+open import Categories.Functor renaming (id to idF)
+open import Categories.NaturalTransformation using (NaturalTransformation)
+open import Categories.Monad using (Monad)
+
+open import Data.Nat.Properties renaming (+-0-monoid to +-0-nat-monoid)
+open import Data.Integer.Properties renaming (+-0-monoid to +-0-integer-monoid)
+open import Data.Rational.Properties renaming (+-0-monoid to +-0-rational-monoid)
+
+open Monoid public
+
+monoid×-₀ : Monoid 0ℓ 0ℓ → Setoid 0ℓ 0ℓ → Setoid 0ℓ 0ℓ
+monoid×-₀ record { Carrier = Carrierₘ ; _≈_ = _≈ₘ_ ; isMonoid = isMonoid } record { Carrier = Carrier ; _≈_ = _≈_ ; isEquivalence = isEquivalence } = record
+  { Carrier = Carrierₘ × Carrier
+  ; _≈_ = λ (x₁ , y₁) (x₂ , y₂) → x₁ ≈ₘ x₂ × y₁ ≈ y₂
+  ; isEquivalence = record
+      { refl = IsEquivalence.refl isEquivalenceₘ , IsEquivalence.refl isEquivalence
+      ; sym = λ (fst , snd) → IsEquivalence.sym isEquivalenceₘ fst , IsEquivalence.sym isEquivalence snd
```

Comment:
`fst` and `snd` are misleading local names that make the code harder to read than necessary. Also please open `IsEquivalence` (locally) to avoid things being quite so verbose; the verbosity really obscures the simplicity of the actual instance.  Again, local module names might make that even clearer.